DesignKit/Variants/Default/DefaultColors.swift

```diff
@@ -0,0 +1,56 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import UIKit
+
+/// Default (light) theme colors. Will be a struct when things are more Swifty.
+public class DefaultColors: Colors {
```

Comment:
I think we can use `LightColors` here. Default things are chosen inside the app.

Reply:
Updated.